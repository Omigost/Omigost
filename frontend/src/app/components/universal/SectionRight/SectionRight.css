@import "../../../css/Theme.less";

.SectionRight {
  padding: 1vw;
  padding-left: 19vw;
  position: relative;
  top: -3vw;
  width: 68vw;
}