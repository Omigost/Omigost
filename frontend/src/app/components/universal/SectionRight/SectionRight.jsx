import React from 'react'
import './SectionRight.css'

class SectionRight extends React.Component {
  render() {
   
    const { children } = this.props;
   
    return (
      <div styleName='SectionRight'>
        {children}
      </div>
    )
  }
}

export default SectionRight