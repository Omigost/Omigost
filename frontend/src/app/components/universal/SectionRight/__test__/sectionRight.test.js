import React from 'react';
import renderer from 'react-test-renderer';

import SectionRight from '../SectionRight.jsx';

test('Snapshot test of SectionLeft', () => {
    const component = renderer.create(
        <SectionRight>
            <div>A</div>
        </SectionRight>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});