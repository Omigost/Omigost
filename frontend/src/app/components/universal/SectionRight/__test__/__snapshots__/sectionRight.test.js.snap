// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test of SectionLeft 1`] = `
<div
  styleName="SectionRight"
>
  <div>
    A
  </div>
</div>
`;
